tutorials/asr/README.md

```diff
@@ -28,6 +28,8 @@ In this repository, you will find several tutorials discussing what is Automatic
 
 10) `Self_Supervised_Pre_Training`: It can often be difficult to obtain labeled data for ASR training. In this tutorial, we demonstrate how to pre-train a speech model in an unsupervised manner, and then fine-tune with CTC loss.
 
+11) `Offline_ASR_with_VAD_for_CTC_models`: In this tutorial, we will demonstrate how to use offline VAD to extract speech segments and transcribe the speech segments with CTC models. This will help to exclude some non_speech utterances and could save computation resources by removing unnecessary input to the ASR system.
```

Comment:
Create another section called ASR with VAD instead of adding to current list. Then the table of content can quickly jump to it. We will add future RNNT too so might as well.

Reply:
Actually, I don't think we need to add another section.  RNNT will go to this tutorial by removing CTC models. And streaming vad asr will go to 

> Streaming / Buffered Automatic Speech Recognition: Streaming_ASR_with_VAD_for_CTC_models